### ISSUE TITLE
Bold weight

### ISSUE THREAD

**Issue by DivineDominion** [quoteunquoteapps/CourierPrimeSource#2]
Bold font weight would be super useful for syntax highlighting in code editors!

**Comment by dwiyatci** [comment]
🆙

**Comment by brdjns** [comment]
I agree. Many editor themes also use bold for highlighting, which unfortunately shows up as simulated bold when using Courier Prime Code. Any chance of a bold (and maybe bold italic) variant being created?
